packages/data-provider/src/repository/submittedRepository.ts

```diff
@@ -1,21 +1,31 @@
-import { SQL, and, count, eq } from 'drizzle-orm/sql';
+import { SQL, and, count, eq, inArray, isNull, or, sql } from 'drizzle-orm/sql';
 
 import { NewSubmittedData, SubmittedData, submittedData } from 'data-model';
 import { BaseDependencies } from '../config/config.js';
 import { ServiceUnavailable } from '../utils/errors.js';
-import { BooleanTrueObject, PaginationOptions, SubmittedDataRepository } from '../utils/types.js';
+import { BooleanTrueObject, PaginationOptions, SubmittedDataResponse } from '../utils/types.js';
 
 const repository = (dependencies: BaseDependencies) => {
 	const LOG_MODULE = 'SUBMITTEDDATA_REPOSITORY';
 	const { db, logger } = dependencies;
 
+	// Column name on the database used to build JSONB query
+	const jsonbColumnName = 'data';
```

Comment:
this (as with most constants) could be abstracted into an env var

Reply:
this value is now imported from the `data-model`